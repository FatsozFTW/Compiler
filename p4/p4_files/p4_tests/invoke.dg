int f(int arg1,bool arg2){
}
void g(){
	int a;
	f(a, true);
	g();
	bool aRandomBool;
	if(g <= (1+a or aRandomBool))
	{
		output true;
		return true;
	}
	else
	{
		int n;
		for(int i; i<n; i++)
		{
			i = n+i;
			aRandomBool = !aRandomBool;
			f(a, true);

		}
		return false;
		while(aRandomBool)
        {
            g();
			f(a, true);
        }
	}
	
}
