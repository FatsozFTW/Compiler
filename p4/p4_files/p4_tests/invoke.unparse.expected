int f(int,bool->int)(int arg1(int), bool arg2(bool)){
}
void g(->void)(){
    int a(int);
    f(int,bool->int)(a(int), true);
    g(->void)();
}

